packages/venia-concept/yarn-error.log

```diff
@@ -0,0 +1,12112 @@
+Arguments: 
```

Comment:
something tells me we want to add this file to the `.gitignore`

Reply:
Correct, and done. 👍